bundles/core/org.eclipse.smarthome.core.voice/src/main/java/org/eclipse/smarthome/core/voice/VoiceManager.java

```diff
@@ -492,6 +513,26 @@ public STTService getSTT() {
     }
 
     /**
+     * Retrieves a STT service with the given id.
+     *
+     * @param id the id of the STT service
+     *
```

Comment:
see above, I will not add any additional comments about the empty line between @param and @return (and other tags).